src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/OsmFileParserCommand.java

```diff
@@ -0,0 +1,68 @@
+package org.openstreetmap.atlas.utilities.command.subcommands;
+
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.utilities.command.AtlasShellToolsException;
+import org.openstreetmap.atlas.utilities.command.abstractcommand.AbstractAtlasShellToolsCommand;
+import org.openstreetmap.atlas.utilities.command.abstractcommand.OptionAndArgumentDelegate;
+import org.openstreetmap.atlas.utilities.command.parsing.ArgumentArity;
+import org.openstreetmap.atlas.utilities.command.parsing.ArgumentOptionality;
+import org.openstreetmap.atlas.utilities.testing.OsmFileParser;
+
+/**
+ * @author matthieun
+ */
+public class OsmFileParserCommand extends AbstractAtlasShellToolsCommand
+{
+    private static final String JOSM_OSM_FILE = "josm";
+    private static final String OSM_FILE = "osm";
+
+    private final OptionAndArgumentDelegate optionAndArgumentDelegate;
+
+    public static void main(final String[] args)
+    {
+        new OsmFileParserCommand().runSubcommandAndExit(args);
+    }
+
+    public OsmFileParserCommand()
+    {
+        this.optionAndArgumentDelegate = this.getOptionAndArgumentDelegate();
+    }
+
+    @Override
+    public int execute()
+    {
+        final String josmOsmFile = this.optionAndArgumentDelegate.getUnaryArgument(JOSM_OSM_FILE)
+                .orElseThrow(AtlasShellToolsException::new);
+        final String osmFile = this.optionAndArgumentDelegate.getUnaryArgument(JOSM_OSM_FILE)
+                .orElseThrow(AtlasShellToolsException::new);
+        new OsmFileParser().update(new File(josmOsmFile), new File(osmFile));
+        return 0;
+    }
+
+    @Override
+    public String getCommandName()
+    {
+        return "josm-to-osm";
+    }
+
+    @Override
+    public String getSimpleDescription()
+    {
+        return "Transform a JOSM OSM file into a real OSM file.";
+    }
+
+    @Override
+    public void registerManualPageSections()
+    {
```

Comment:
Please add at least some simple `DESCRIPTION` and `EXAMPLES` sections. [See here for an example of how to call the API.](https://github.com/osmlab/atlas/blob/ac6808336181f80e2c12b411506d1cab20de8a08/src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/IsoCountryCodeCommand.java#L131) [See here for an example DESCRIPTION.](https://github.com/osmlab/atlas/blob/dev/src/main/resources/org/openstreetmap/atlas/utilities/command/subcommands/IsoCountryCodeCommandDescriptionSection.txt) [See here for an example of EXAMPLES.](https://github.com/osmlab/atlas/blob/dev/src/main/resources/org/openstreetmap/atlas/utilities/command/subcommands/IsoCountryCodeCommandExamplesSection.txt)

Reply:
:)